src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,49 @@
+package mate.controller;
```

Comment:
create a separate packages driver, car, manufacturer in controller package and divide existing controllers between them.